BPCompatibleAlertControllerDemo/CompatibleAlertControllerDemo/ViewController.swift

```diff
@@ -9,52 +9,69 @@
 import UIKit
 
 class ViewController: UIViewController {
-    var alertController: BPCompatibleAlertController?
 
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        var observerObject: NSObjectProtocol?
+        
+        let cleanupAfterAlertDismissed: (Void) -> Void =
```

Comment:
OK. I'll add something like that and let you know when it's in the pull request for review.